external/solr/src/main/java/com/digitalpebble/storm/crawler/solr/persistence/StatusUpdaterBolt.java

```diff
@@ -79,6 +80,9 @@ public void store(String url, Status status, Metadata metadata,
         SolrInputDocument doc = new SolrInputDocument();
 
         doc.setField("url", url);
+
+        doc.setField("host", URLUtil.getHost(url));
```

Comment:
I'd rather we did that properly later + have you modified the schema files accordingly?